src/net_processing.cpp

```diff
@@ -2603,8 +2604,8 @@ bool static ProcessMessage(CNode* pfrom, const std::string& strCommand, CDataStr
             LogPrint(BCLog::MEMPOOLREJ, "%s from peer=%d was not accepted: %s\n", tx.GetHash().ToString(),
                 pfrom->GetId(),
                 FormatStateMessage(state));
-            if (enable_bip61 && state.GetRejectCode() > 0 && state.GetRejectCode() < REJECT_INTERNAL) { // Never send AcceptToMemoryPool's internal codes over P2P
-                connman->PushMessage(pfrom, msgMaker.Make(NetMsgType::REJECT, strCommand, (unsigned char)state.GetRejectCode(),
+            if (enable_bip61 && state.IsInvalid()) {
```

Comment:
In commit "[validation] Remove Reject Code from ValidationState" (a3076817588a642b4de107dc96a76ea8939070df):

Note: IIUC, there is no change in behavior here because previous `state.GetRejectCode() < REJECT_INTERNAL` condition was always true. REJECT_HIGHFEE is never triggered for p2p transactions.

Reply:
No longer part of this PR.